### PR TITLE
Android Studio 4.0

### PR DESCRIPTION
**Pull request by hannesa2**

### CODE REVIEW

**Comment by cketti** [comment]
Thanks. I gave this a quick spin on an API 21 emulator and got a crash at startup.

It appears that in order to be able to use Java 8 libraries on API versions that don't support Java 8 APIs "library desugaring" has to be enabled. See https://developer.android.com/studio/releases/gradle-plugin#j8-library-desugaring

Can you please update the pull request with the required changes to `build.gradle` and make sure it runs fine on an API 21 emulator (and maybe API 23, because I've heard of some issues related to Java 8, too)?

**Comment by hannesa2** [comment]
I tried to fix the Espresso tests, but few remaining are in a very bad condition https://github.com/k9mail/k-9/pull/4817

But a smoke test would make sense for each pull request

File: build.gradle [hunk 1]
```diff
@@ -25,13 +25,14 @@ buildscript {
                 'androidxPreference': '1.1.1',
                 'materialComponents': '1.1.0',
                 'preferencesFix': '1.1.0',
-                'okio': '2.5.0',
+                'okio': '2.6.0',
+                'desugar_jdk_libs': '1.0.5',
```
Comment (cketti) [inline build.gradle:29]: For consistency can you please change this to use camel case?

Comment (hannesa2) [reply build.gradle:29]: I've no clue what's the next urgent wish. I recommend to rename it after a merge by yourself, then it's exactly like you want

**Pull request closed without merging**